conf/app.routes

```diff
@@ -1,6 +1,7 @@
 # microservice specific routes
 
 GET         /eori/:eori/verified-email                  controllers.VerifiedEmailController.getVerifiedEmail(eori: String)
+GET         /eori/verified-email                        controllers.VerifiedEmailController.getVerifiedEmailV2()
```

Comment:
Better to give a line break after this line to keep both these similar routes together and to differentiate it from rest of the routes